server/data/dr.strange.json.json

```diff
@@ -1,6 +1,6 @@
 {
   "created": 1647693129833,
-  "currentStrandPath": "",
+  "currentStrandPath": "test strand path",
```

Comment:
This can be left as empty because the value will not be shown on the client side